examples/kv_store/Cargo.toml

```diff
@@ -2,18 +2,15 @@
 name = "kv_store"
 version = "0.1.0"
 edition = "2021"
+default-run = "kv_store"
```

Comment:
is this needed?

Reply:
its needed if we want to run the `kv_store` by simply `cargo run`. Otherwise we need to type `cargo run --bin kv_store` . I can add some comments in the code if you want, or just remove this line?

Reply:
remove this line. If you want to run kv_store you change directory to kv_store and then do `cargo_run`